src/lib.rs

```diff
@@ -17,7 +17,7 @@
 #![feature(core_intrinsics, core_slice_ext, core_str_ext)]
 #![allow(improper_ctypes)]
 #![feature(const_fn)]
-#![deny(missing_docs)]
+// #![deny(missing_docs)]
```

Comment:
I agree, fixed docs is the best solution here. It fails on most of the `const` definitions in the `tiva_c` platform tree code which probably don't need documenting. For example:

```
pub const PORT_A: *const Port = 0x40004000 as *const Port;
pub const PORT_B: *const Port = 0x40005000 as *const Port;
pub const PORT_C: *const Port = 0x40006000 as *const Port;
```

in `/src/hal/tiva_c/pin.rs`. I can add `#[allow(missing_docs)]` everywhere, but it seems very messy:

```
#[allow(missing_docs)] pub const PORT_A: *const Port = 0x40004000 as *const Port;
#[allow(missing_docs)] pub const PORT_B: *const Port = 0x40005000 as *const Port;
#[allow(missing_docs)] pub const PORT_C: *const Port = 0x40006000 as *const Port;
```

Should I just add `#[allow(missing_docs)]` and uncomment the line in `src/lib.rs` or is there a better way?